src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -215,65 +216,80 @@
-            fiat-limit                (.toFixed (* token-balance conversion-rate) 2)
-            current-limit             #(if @crypto-currency? crypto-limit fiat-limit)
-            current-currency          (if @crypto-currency? token-symbol fiat-currency)
-            limit-label               (make-limit-label {:amount   (current-limit)
-                                                         :currency current-currency})
-            input-num-value           (parse-double @input-value)
-            confirm-disabled?         (or (nil? route)
-                                          (empty? route)
-                                          (empty? @input-value)
-                                          (<= input-num-value 0)
-                                          (> input-num-value (current-limit)))
-            amount-text               (str @input-value " " token-symbol)
-            native-currency-symbol    (when-not confirm-disabled?
-                                        (get-in route [:from :native-currency-symbol]))
-            native-token              (when native-currency-symbol
-                                        (rf/sub [:wallet/token-by-symbol native-currency-symbol]))
-            fee-in-native-token       (when-not confirm-disabled?
-                                        (send-utils/calculate-full-route-gas-fee route))
-            fee-in-crypto-formatted   (when fee-in-native-token
-                                        (utils/get-standard-crypto-format native-token
-                                                                          fee-in-native-token))
-            fee-in-fiat               (when-not confirm-disabled?
-                                        (utils/calculate-token-fiat-value
-                                         {:currency fiat-currency
-                                          :balance  fee-in-native-token
-                                          :token    native-token}))
-            currency-symbol           (rf/sub [:profile/currency-symbol])
-            fee-formatted             (when fee-in-fiat
-                                        (utils/get-standard-fiat-format fee-in-crypto-formatted
-                                                                        currency-symbol
-                                                                        fee-in-fiat))
-            show-select-asset-sheet   #(rf/dispatch
-                                        [:show-bottom-sheet
-                                         {:content (fn []
-                                                     [select-asset-bottom-sheet clear-input!])}])]
+      (let [{fiat-currency :currency}        (rf/sub [:profile/profile])
+            {token-balance            :total-balance
+             token-symbol             :symbol
+             token-networks           :networks
+             token-balances-per-chain :balances-per-chain
```

Comment:
actually this is something we should avoid, 

please see my pr here:

https://github.com/status-im/status-mobile/pull/19568

Basically the issue here is that this file is now creating two sets of the same data, one in the account and one in the send data model. It is better to keep one source of truth so we should store the minimum here and instead look up the account balances based off the send token symbol.

Reply:
cc @ulisesmac as it's something to consider about the data model. We should probably be sure to watch out for this in other places where we can be duplicating some parts of the db

Reply:
I see, and I agree. Maybe we can filter `:balances-per-chain` and other sensible props from `:wallet/wallet-send-token` sub to avoid unwanted bugs.

Anyways, just updated the code to use `:balances-per-chain` from the same sub of your PR 👍